README.md

```diff
@@ -1,95 +1,94 @@
 # AtlasSupportSDK
 
-**AtlasSupportSDK** is a mobile SDK that integrates a real-time chat widget into iOS applications. 
+[![Version](https://img.shields.io/cocoapods/v/AtlasSupportSDK.svg?style=flat)](https://cocoapods.org/pods/AtlasSupportSDK)
+[![License](https://img.shields.io/cocoapods/l/AtlasSupportSDK.svg?style=flat)](https://cocoapods.org/pods/AtlasSupportSDK)
+[![Platform](https://img.shields.io/cocoapods/p/AtlasSupportSDK.svg?style=flat)](https://cocoapods.org/pods/AtlasSupportSDK)
 
----
+
+**AtlasSupportSDK** is a mobile SDK that integrates a real-time chat widget into iOS applications. 
 
 ## Installation
 
+You can add Swift SDK to your application in two ways: via **SPM** or **CocoaPods**.
+
 ### SPM
 
-1. Open Xcode & Add the SDK
-2. Open your Xcode project.
-3. Go to File → Add Packages.
-4. In the search bar, enter the repository URL of the SDK:
+1. In the Xcode, when your project is opened, select **File → Add Package Dependencies...**
```

Comment:
I would prefer 'Open your Xcode project and select **File → Add Package Dependencies...**'